src/recommendations/data/hooks/useStaticRecommendations.jsx

```diff
@@ -4,7 +4,12 @@ import { getConfig } from '@edx/frontend-platform';
 
 import { filterLocationRestriction } from '../utils';
 
-export default function useProducts(countryCode) {
+/**
+ * This hooks returns Static recommendations (Trending and Popular products) after applying location restrictions.
+ * @param countryCode
+ * @returns {{isLoading: boolean, trendingProducts: *[], popularProducts: *[]}}
+ * */
+const useStaticRecommendations = (countryCode) => {
   const [isLoading, setLoading] = useState(true);
```

Comment:
Do we need `isLoading` check in this hook?

Reply:
yes we need it, because the in first render the products list is empty and our recommendations page get redirected to next url.